### ISSUE TITLE
Add support for Interior-Point method for GLPK

### ISSUE THREAD

**Issue by synchon** [biosustain/optlang#174]
GLPK supports Interior-Point method for LP solution. GX-FBA (a potential method for adding in driven) requires this method explicitly for it's implementation. While the CPLEX and Gurobi interface already provides it, it would be nice to have the interface for GLPK working too.

**Comment by KristianJensen** [comment]
I agree, that would definitely be nice to have. It should also be fairly simple to implement as far as I can see.

**Comment by synchon** [comment]
Thank you for your reply! If it's not on your to-do checklist for now, let me know if I should open a PR. :D

**Comment by KristianJensen** [comment]
If you want to try to implement it, that would be great :)
